controller/api/public/gateways.go

```diff
@@ -85,10 +84,6 @@ func processPrometheusResult(results []promResult) map[string]*pb.GatewaysTable_
 				} else {
 					rows[key].Alive = true
 				}
```

Comment:
The CLI user may have RBAC permissions to interact with the `linkerd` namespace, but not necessarily with the `linkerd-multicluster` namespace (say this is not the same user that installed the mc controller). The controller on the other hand has guaranteed cluster access, so it might make more sense to keep this in the controller.

Reply:
Also this could become a one liner:
```go
rows[key].Alive = value > 0
```